redash/settings.py

```diff
@@ -134,6 +134,7 @@ def all_settings():
 SESSION_COOKIE_SECURE = parse_boolean(os.environ.get("REDASH_SESSION_COOKIE_SECURE") or str(ENFORCE_HTTPS))
 
 LOG_LEVEL = os.environ.get("REDASH_LOG_LEVEL", "INFO")
+LOG_STREAM = os.environ.get("REDASH_LOG_STREAM", "STDERR")
```

Comment:
Because it's either `stdout` or `stderr`, let's make it simpler -- how about we change the configuration value to a boolean named `REDASH_LOG_STDOUT`?